components/infra-proxy-service/server/roles.go

```diff
@@ -20,23 +20,11 @@ type RunList []string
 // EnvRunList represents the recipes and roles with environment specified for a node or as part of a role.
```

Comment:
RunList and EnvRunList are defined under chef package. We may not need to redefine it here and refer the type defined in chef package.